src/index.js

```diff
@@ -655,6 +659,11 @@ const handleNewChain = (chainId) => {
 
 const handleNewNetwork = (networkId) => {
   networkDiv.innerHTML = networkId;
+  if (networkId === ('11155111' || '0xaa36a7')) {
```

Comment:
Can we make this more readable? Was wondering if we could leverage the `NETWORKS_BY_CHAIN_ID` const.. but might not be worth the effort. We could do something simpler like:
```js
const isNetworkIdSepolia = networkId === ('11155111' || '0xaa36a7');

if (isNetworkIdSepolia) {
  ...
}
```